blob/src/main/java/com/bazaarvoice/emodb/blob/db/astyanax/AstyanaxStorageProvider.java

```diff
@@ -201,14 +200,14 @@ public void deleteObject(Table tbl, String blobId, Integer chunkCount) {
 
     @ParameterizedTimed(type="AstyanaxStorageProvider")
     @Override
-    public void writeMetadata(Table tbl, String blobId, StorageSummary summary, Duration ttl) {
+    public void writeMetadata(Table tbl, String blobId, StorageSummary summary) {
         AstyanaxTable table = (AstyanaxTable) checkNotNull(tbl, "table");
         for (AstyanaxStorage storage : table.getWriteStorage()) {
             BlobPlacement placement = (BlobPlacement) storage.getPlacement();
 
             MutationBatch mutation = placement.getKeyspace().prepareMutationBatch(CONSISTENCY_STRONG)
                     .setTimestamp(summary.getTimestamp());
-            Integer ttlSeconds = Ttls.toSeconds(ttl, 1, null);
+            Integer ttlSeconds = Ttls.toSeconds(null, 1, null);
```

Comment:
There is a signature of `putColumn()` that includes no TTL. We should be able to remove this code altogether!

Reply:
good catch, will do